package/metadata.json

```diff
@@ -0,0 +1,37 @@
+{
+	"KPackageStructure": "Plasma/Applet",
+	"KPlugin": {
+		"Authors": [
+                       {
+                               "Email": "zrenfire@gmail.com",
+                               "Name": "Chris Holland"
+                       },
+                       {
+                               "Email": "aricaldeira@gmail.com",
+                               "Name": "Ari Caldeira"
+                       },
+			{
+				"Email": "oded@geek.co.il",
+				"Name": "Oded Arbel"
+			}
+		],
+		"Category": "System Information",
+		"Description": "Run a command periodically and render its output.",
+		"Description[bz]": "Ezekuta un komandu a kada X milisegundus, i mòstra a saida/rezuwtadu",
+		"Description[fr]": "Exécute une commande périodiquement et affiche son retour.",
+		"Description[nl]": "Voer een opdracht periodiek uit en toon de uitvoer ervan.",
+		"Description[pt_br]": "Executa um comando a cada X milissegundos, e mostra a saída/resultado",
+		"Icon": "utilities-terminal",
+		"Id": "com.github.aricaldeira.commandoutput",
```

Comment:
I know you sorta forked it but if your merging it back upstream we probably shouldn't alter the namespace. Especially if this is suppose to be a drop in replacement from plasma5 => plasma6 since the namespace change will break existing widgets.